lib/utils/addon-utils.ts

```diff
@@ -85,3 +89,25 @@ export function conflictsWith(...addOns: string[]) {
     return descriptor;
   };
 }
+
+/**
+ * Iterate an object to normalize the string format before sending to helm.
+ * For example, escaping the dot from certain keys: "ingress.annotations.kubernetes\\.io/tls-acme"
+ * @returns Values
+ * @param obj: Values
+ */
+export function normalizeValues(obj: Values): Values {
```

Comment:
This looks identical to the function defined in the application.ts under argocd. Why are we duplicating it? If you need to refactor then we can discuss.

Reply:
Needed to use the normalize values to ingest backstage annotations from values. Extracted it from the argocd addon, and instead of purely refactoring, just copied it to utils.